ext/standard/proc_open.c

```diff
@@ -913,7 +985,7 @@ PHP_FUNCTION(proc_open)
 #ifdef PHP_WIN32
 	if (other_options) {
 		suppress_errors      = get_option(other_options, "suppress_errors");
-		/* TODO Deprecate in favor of array command? */
+		/* TODO: Deprecate in favor of array command? */
```

Comment:
I think this might better be discussed on internals@. AFAIK, bypass_shell=true is default for Symfony, were the deprecation could be a big issue. @nicolas-grekas?

Reply:
we would work around the deprecation, no worries on this side